SentinelOne/sentinelone/ingest/parser.yml

```diff
@@ -802,17 +811,17 @@ stages:
           # agentIpV4, agentIpV6 can contain multiples values separated by comma
           host.ip: >
             [
-            {% if json_event.message.agentDetectionInfo.agentIpV4 != null %}
+            {% if json_event.message.agentDetectionInfo.agentIpV4 | is_ipaddress %}
               {% for ip in json_event.message.agentDetectionInfo.agentIpV4.split(',') %}
                 '{{ip}}',
               {% endfor %}
             {% endif %}
-            {% if json_event.message.agentDetectionInfo.agentIpV6 != "" %}
+            {% if json_event.message.agentDetectionInfo.agentIpV6 | is_ipaddress %}
               {% for ip in json_event.message.agentDetectionInfo.agentIpV6.split(',') %}
                 '{{ip}}',
               {% endfor %}
             {% endif %}
-            {% if json_event.message.agentDetectionInfo.externalIp != null %}
+            {% if json_event.message.agentDetectionInfo.externalIp | is_ipaddress %}
```

Comment:
These changes does not work as intended.

Reply:
What does not work as intended?

Reply:
The ips that should be produced by this code are deleted in the updated tests.
You are applying the filter on a string, not on an array. You can see the next line is splitting the string to make the array. I tried to fix it on my own with an `{% if ip | is_ipaddress %}` but it did not seem to work either.

Reply:
I see thank you !=)

Reply:
This issue should now be fixed, thank you for taking the time to review my work :+1: